File: java/gltfio/src/main/java/io/github/erkko68/filament/gltfio/Gltfio.java
package io.github.erkko68.filament.gltfio;

import io.github.erkko68.filament.Filament;

public class Gltfio {
    public static void init() {
        Filament.init();
        System.loadLibrary("gltfio-jni");
    }
}

File: kotlin/gltfio/src/commonMain/kotlin/io/github/erkko68/filament/gltfio/Gltfio.kt
package io.github.erkko68.filament.gltfio

expect object Gltfio {
    /**
     * Initializes the gltfio JNI layer on Android and sets up any necessary
     * native state on other platforms. Must be called before using any
     * gltfio functionality.
     */
    fun init()
}
